src/quo2/components/messages/author/view.cljs

```diff
@@ -0,0 +1,65 @@
+(ns quo2.components.messages.author.view
+  (:require
+   [quo2.components.messages.author.style :as style]
+   [quo.react-native :as rn]
+   [quo2.components.markdown.text :as text]
+   [quo2.components.icon :as icons]
```

Comment:
[Use quo2.core](https://github.com/status-im/status-mobile/blob/chore/document-conventions-and-guidelines/doc/new-guidelines.md#requiring-quo2-components)

Reply:
I think we can keep it inside `quo2` based on previous conversations https://github.com/status-im/status-mobile/pull/14304#discussion_r1019352852
Also if we are making an exception for this rule inside quo2 maybe would be fine to update the guideline? cc @ilmotta

Reply:
:+1: I'm going to write this addendum